site/ru/blog/new-in-devtools-104/index.md

```diff
@@ -5,184 +5,233 @@ authors:
+## Предпросмотр нецветных значений при автодополнении переменных CSS {: #css-var }
 
 <!-- When autocompleting CSS variables, DevTools now populates the non-color variable with a meaningful value so that you can preview what kind of change the value will have on the node. -->
+Когда CSS-переменная автоматически дополняется, DevTools теперь заполняет «нецветную» переменную осмысленным значением, чтобы вы могли предварительно просмотреть, как изменится значение на узле.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/V4slwNtX9HwLPdAyr8JF.png", alt="Preview non-color value when autocompleting CSS variables", width="800", height="431" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/V4slwNtX9HwLPdAyr8JF.png", alt="редпросмотр нецветных значений при автодополнении переменных CSS", width="800", height="431" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/977cc58cb5654a2b68142ef8ac1b3f9ac2822694 #}
 
-Chromium issue: [1285091](https://crbug.com/1285091)
+Задача в трекере Chromium: [1285091](https://crbug.com/1285091)
 
         
 <!-- ## Identify blocking frames in the Back/forward cache pane {: #bfcache } -->
+## Определение блокирующего фрейса на панели Возвратного кэша (Back/forward cache) {: #bfcache }
 
 <!-- The [Back/forward cache](/docs/devtools/application/back-forward-cache/) pane in the **Application** panel has new **frames** section to help you identify blocking frames that may be preventing the page from being eligible for bfcache. -->
+Панель [Возвратный кэш (Back/forward cache)](/docs/devtools/application/back-forward-cache/) во вкладке **Приложение** (Application) теперь содержит раздел **Фреймы** (frames). Это поможет определить блокирующие фреймы, которые могут препятствовать загрузке страницы из кэша.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/UaRYEoYYoXhjSIn9seYK.png", alt="Identify blocking frames in the Back/forward cache pane", width="800", height="486" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/UaRYEoYYoXhjSIn9seYK.png", alt="Определение блокирующего фрейса на панели Возвратного кэша (Back/forward cache)", width="800", height="486" %}
  
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/897799b24fff0639d483111dd2d957288ba2bd06 #}
  
-Chromium issue: [1288158](https://crbug.com/1288158) 
+Задача в трекере Chromium: [1288158](https://crbug.com/1288158) 
  
  
 <!-- ## Improved autocomplete suggestions for JavaScript objects {: #autocomplete } -->
+## Улучшены подсказки автодополнения для объектов JavaScript {: #autocomplete }
 
 <!-- The the autocompletion for JavaScript object properties now display based on this order: -->
+Подсказки автодополнения для свойств объектов JavaScript теперь отображаются в следующем порядке:
 
 <!-- 1. Own enumerable properties
 2. Own non-enumerable properties
 3. Inherited enumerable properties
 4. Inherited non-enumerable properties -->
+1. Собственные перечисляемые свойства;
```

Comment:
Will this show up as 1. 2. 3. 4.?

Reply:
Yes, Mrkdown can render it as 1, 2, 3. It's _lazy_ list =)